src/Components/test/testassets/ComponentsApp.Server/Startup.cs

```diff
@@ -35,7 +36,10 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseEndpoints(endpoints =>
             {
                 endpoints.MapRazorPages();
-                endpoints.MapComponentHub<App.App>("app");
+                endpoints.MapControllers();
+                endpoints.MapHub<ComponentHub>("/dynamic/_blazor");
```

Comment:
What's this for?

Reply:
Needed two different hubs, one for the "statically registered app component" and the other for the one registered via prerendering.